CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+ComplianceAsCode Contribution Guidelines
+===
+
+We're so glad you're thinking about contributing to ComplianceAsCode! If you're unsure about anything, just ask -- or submit the issue or pull request to start the conversation! The worst that could happen is you'll politely be asked to change something.
+
+We want to ensure a welcoming environment for all members of the ComplianceAsCode community. Reading the projects CONTRIBUTING policy (you are here), its [LICENSE](https://github.com/ComplianceAsCode/content/blob/master/LICENSE), and its [README](https://github.com/ComplianceAsCode/content/blob/master/README.md), will help you get calibrated.
+
+This project is a public-private partnership with the United States Government as part of its [open source initiatives](https://code.nsa.gov/). The project has welcomed contributions from across the United States Government, private industry, and countries worldwide. Currently, there are four project maintainers who are accountable to ensure a welcoming community and our code of conduct is followed. Should you ever need, please feel free to reach out directly!
+
+- @shawndwells (Shawn Wells // CrowdStrike // shawn.wells@crowdstrike.com)
+- @redhatrises  (Gabe Alford // Red Hat // galford@redhat.com)
+- @jeffblank (Jeff Blank // U.S. Department of Defense // jeffrey.d.blank@ugov.gov)
+- @carlosmmatos (Carlos Matos // Red Hat // cmatos@redhat.com)
```

Comment:
This list of four names might be seen as an exhaustive list of persons, who could decide the fate of contributions to the project or provide guidance (again, it is not a generic README file, this guideline is all about contributions). Is it the impression you want to communicate?

Reply:
The project has much more maintainers and everyone should be responsible for following this document therefore there is no need to pick 4 of them and list them explicitly.

Reply:
While true to some degree, the project maintainers here are ultimately responsible to the US government as trusted stewards of the private relationship. They are the ones held accountable and are able to meet directly to address concerns and issues over the project including what gets merged.

Reply:
So... as a non-US citizen, I can't become a "maintainer"? Is this what is being conveyed here? The list of maintainers and the paragraph above this just made this whole document very confusing.
What does being a "maintainer" mean for this project then? Might want to start defining that as well.

Reply:
Good point. That's an incorrect assumption which is hopefully clarified now.